### ISSUE TITLE
Password might not contain all the different character types !!

### ISSUE THREAD

**Issue by study-avishek** [FantoX001/Password-Generator#1]
As it is only using one variable "extendedString" to store all the characters and symbols, it might happen that the generated password is not randomized and it might not contain all the different type of characters .

**Comment by FantoX001** [comment]
Yes i will recreate a 2nd version of this software to fix those problems. Thanks for your cooperation
